org.eclipse.tm4e.ui/src/main/java/org/eclipse/tm4e/ui/internal/TMUIMessages.properties

```diff
@@ -18,6 +18,7 @@ Button_browse_Workspace=Browse Workspace...
 
 # preference page
 TextMatePreferencePage_GrammarRelatedLink=See <a>''{0}''</a> for associating editors with grammars.
+TextMatePreferencePage_LanguageConfigurationRelatedLink=See <a>''{0}''</a> for associating editors with language configurations.

```

Comment:
It seems this TextMatePreferencePage_LanguageConfigurationRelatedLink is not used?

Reply:
Added the rest of the logic to have a button pointing the user to the preference page